R/projectR.R

```diff
@@ -82,60 +71,61 @@ setMethod("projectR",signature(data="matrix",loadings="matrix"),function(
 #' @param NP vector of integers indicating which columns of loadings object to use. The default of NP=NA will use entire matrix.
 #' @param full logical indicating whether to return the full model solution. By default only the new pattern object is returned.
 #' @param model Optional arguements to choose method for projection
-#' @param family VGAM family function for model fitting (default: "gaussianff")
 #' @rdname projectR-methods
 #' @aliases projectR
 setMethod("projectR",signature(data="dgCMatrix",loadings="matrix"),function(
   data, # a dataset to be projected onto
   loadings, # a matrix of continous values to be projected with unique rownames
   dataNames = NULL, # a vector with names of data rows
   loadingsNames = NULL, # a vector with names of loadings rows
-  NP=NA, # vector of integers indicating which columns of loadings object to use. The default of NP=NA will use entire matrix.
-  full=FALSE, # logical indicating whether to return the full model solution. By default only the new pattern object is returned.
-  family="gaussianff"  # VGAM family function (default: "gaussianff")
+  NP=NULL, # vector of integers indicating which columns of loadings object to use. The default of NP=NA will use entire matrix.
```

Comment:
update comment to match new default

Reply:
also, it's noteworthy to me that the default for `NP` is `NA` above and `NULL` here. Not sure if it's weird or wrong. just noting it

Reply:
yup, because `is.na(NP)` was never going to work if NP is specified:
```
> NP <- c(1,3,5)
> if(!is.na(NP)) {print("it's a bug")}
Error in if (!is.na(NP)) { : the condition has length > 1
```
is.null works fine:
```
> is.null(NP)
[1] FALSE
```
see line 85 too - I changed the check from `is.NA` to `is.null` too